arithmetization/src/main/java/net/consensys/linea/bl/TransactionProfitabilityCalculator.java

```diff
@@ -25,68 +25,58 @@
 
 @Slf4j
 public class TransactionProfitabilityCalculator {
-
+  private static final long TO_WEI_MULTIPLIER = 1_000;
   private final LineaProfitabilityConfiguration profitabilityConf;
-  private final double preComputedValue;
-  private final double priceAdjustment;
 
   public TransactionProfitabilityCalculator(
       final LineaProfitabilityConfiguration profitabilityConf) {
     this.profitabilityConf = profitabilityConf;
-    this.preComputedValue =
-        profitabilityConf.gasPriceRatio() * profitabilityConf.verificationGasCost();
-    this.priceAdjustment = profitabilityConf.gasPriceAdjustment().getAsBigInteger().doubleValue();
   }
 
   public Wei profitablePriorityFeePerGas(
       final Transaction transaction,
       final double minMargin,
-      final Wei minGasPrice,
-      final long gas) {
-    final double compressedTxSize = getCompressedTxSize(transaction);
+      final long gas,
+      final Wei minGasPrice) {
```

Comment:
I think now it becomes important to be explicit that minGasPrice is in Wei, because it isn't obvious by looking at this class alone

Reply:
done